vips/vip-194/vip-194-testnet.ts

```diff
@@ -0,0 +1,210 @@
+  },
+  {
+    name: "vUSDT_LiquidStakedBNB",
+    address: "0x2197d02cC9cd1ad51317A0a85A656a0c82383A7c",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vUSDD_LiquidStakedBNB",
+    address: "0xD5b20708d8f0FcA52cb609938D0594C4e32E5DaD",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vSnBNB_LiquidStakedBNB",
+    address: "0xeffE7874C345aE877c1D893cd5160DDD359b24dA",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vFLOKI_GameFi",
+    address: "0xef470AbC365F88e4582D8027172a392C473A5B53",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vUSDD_GameFi",
+    address: "0xdeDf3B2bcF25d0023115fd71a0F8221C91C92B1a",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vRACA_GameFi",
+    address: "0x1958035231E125830bA5d17D168cEa07Bb42184a",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vUSDT_GameFi",
+    address: "0x0bFE4e0B8A2a096A27e5B18b078d25be57C08634",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vUSDT_DeFi",
+    address: "0x80CC30811e362aC9aB857C3d7875CbcCc0b65750",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vankrBNB_DeFi",
+    address: "0xe507B30C41E9e375BCe05197c1e09fc9ee40c0f6",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vUSDD_DeFi",
+    address: "0xa109DE0abaeefC521Ec29D89eA42E64F37A6882E",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vANKR_DeFi",
+    address: "0xb677e080148368EeeE70fA3865d07E92c6500174",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vBSW_DeFi",
+    address: "0x5e68913fbbfb91af30366ab1B21324410b49a308",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vALPACA_DeFi",
+    address: "0xb7caC5Ef82cb7f9197ee184779bdc52c5490C02a",
+    reduceReservesBlockDelta: 100,
+  },
+  {
+    name: "vTWT_DeFi",
+    address: "0x4C94e67d239aD585275Fdd3246Ab82c8a2668564",
+    reduceReservesBlockDelta: 100,
+  },
+];
+
+export const vip194Testnet = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-194 VToken Upgrade of AIA",
+    description: `upgrade the implementation of the Vtoken core supportimg Automatic income allocation feature.`,
+    forDescription: "I agree that Venus Protocol should proceed with VToken Upgrade of AIA",
+    againstDescription: "I do not think that Venus Protocol should proceed with VToken Upgrade of AIA",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds with VToken Upgrade of AIA or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setReduceReservesBlockDelta(uint256)", FAST_TRACK_TIMELOCK],
+      },
+
+      {
+        target: ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setReduceReservesBlockDelta(uint256)", CRITICAL_TIMELOCK],
+      },
```

Comment:
They have already granted in VIP-192

Reply:
[d269d84](https://github.com/VenusProtocol/vips/pull/67/commits/d269d848de1224cb4e714713697b54947136fb3a)